css/css-color/prophoto-rgb-004-ref.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: CSS Color 4: prophoto-rgb</title>
+<style>
+    .test { background-color: lab(87.5745% -186.6921 150.9905); width: 12em; height: 12em; } /* color(prophoto-rgb 0 1 0) converted to Lab */
```

Comment:
So this one is undisplayable because it is slightly outside the spectral locus (see https://drafts.csswg.org/css-color-4/images/UCS-prophoto.svg) but again, the conversion to lab is correct and as the colors are identical they should gamut map the same way